lib/argumentsParser.rb

```diff
@@ -0,0 +1,40 @@
+require "./lib/arguments.rb"
+
+class ArgumentsParser
+  def self.parse()
```

Comment:
Old habits😄. You can omit () in ruby and you should, because this is the norm. Don't be afraid omitting ()

Reply:
Please also review the file name, ruby uses snake case for filenames and variables, and camelcase for class names, [here](https://rubystyle.guide/) are a couple of pointers for style in ruby.